### DISCUSSION TITLE
http/2 CONTINUATION flood impacted ?

### DISCUSSION THREAD

**Discussion by AndyLCQ** [nginx/unit#1239] [Support] [answered]
Hello, 

Does anyone know if Nginx Unit is impacted by the recent http/2 CVE CONTINUATION flood ?
I saw it for some web server as nodejs https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-27983, Envoy, nghttp2 are affected by CVE whereas Haproxy is not.

I try to find any similar discussion here, in Unit's news, and other and found nothing. Does it mean no impact or its has not been tested or identified  ?

Sorry if this question looks stupid :) 

Thank you for support

**Comment by lcrilly** [comment, marked as answer]
Hello! As per https://github.com/nginx/unit/issues/388 Unit does not support HTTP/2 so that's why where's nothing to find :)

**Reply by AndyLCQ** [reply]
Oh right ! 

My fault I thought it was.
Thanks for quick answer.
